content/docs/learning-guide/tutorials/environments/create-terrain-from-images/create-terrain-assets.md

```diff
@@ -0,0 +1,125 @@
+---
+linkTitle: Create Terrain Assets
+title: Create Terrain Assets
+description: Create the terrain assets needed for this tutorial.
+weight: 200
+toc: true
+---
+
+## Goal
+
+In this tutorial section, you will create the terrain assets needed for authoring the example terrain in a level.
+
+## Create a surface tag name list
```

Comment:
I'd prefer to leave this section where it's at, though I agree it's a little bit awkward.

The Surface Tag Name List _is_ an asset, which is why I've put it in the "Create Terrain Assets" tutorial. I put it at the top because it's a more fundamental asset than the rendering assets. It's something that you need and use even if you don't have any rendering in your level at all.

I originally had the asset creation in the "Texture the Terrain" topic, but that topic became extremely bulky and was covering too many concepts at once, which is why I split this page out. 

In particular, I also kept moving around the combination of the "Surface Tag Name List" creation and the "add a Terrain Surface Gradient List" component, since these are both about defining fundamental terrain data, not about rendering. But I put the Terrain Surface Gradient List steps back onto the "Texture the Terrain" page because it's really hard to visualize and understand the surface types without the rendering piece to support it.